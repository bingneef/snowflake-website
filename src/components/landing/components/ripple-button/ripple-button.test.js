import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import RippleButton from "./ripple-button";

describe("RippleButton", () => {
  const defaultProps = {
    onClick: () => null
  };
  it("renders without error", () => {
    const div = document.createElement("div");
    ReactDOM.render(<RippleButton {...defaultProps} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("matches the snapshot", () => {
    const component = renderer.create(<RippleButton {...defaultProps} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
