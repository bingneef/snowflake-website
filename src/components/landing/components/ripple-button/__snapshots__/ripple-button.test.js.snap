// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RippleButton matches the snapshot 1`] = `
<div
  className="ripplePlaceholder"
>
  <a
    className="button is-success button nextButton"
    onClick={[Function]}
  >
    <i
      className="fa fa-lg fa-arrow-down"
    />
  </a>
  <div
    className="ripple"
  />
  <div
    className="rippleDelay"
  />
</div>
`;
